### PR TITLE
Entropy chart now uses real data. (/m interface)

### PR DESCRIPTION
**Pull request by iriahi**
This is a very simple first PR regarding the /m interface.
I will open an issue in the radare2 repository to display real data with pie charts.

### CODE REVIEW

File: www/m/index.html [hunk 1]
```diff
@@ -244,34 +240,8 @@ <h2 class="mdl-card__title-text">Fortunes</h2>
       <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 500 250" style="position: fixed; left: -1000px; height: -1000px;">
         <defs>
           <g id="chart">
-            <g id="Gridlines">
```
Comment (radare) [inline www/m/index.html]: i hope those graphs come back :)

File: www/m/index.js [hunk 2]
```diff
@@ -1091,6 +1090,34 @@ function updateInfo() {
 	});
 }
 
+function updateEntropy() {
+	r2.cmd ("p=", function(d) {
```
Comment (radare) [inline www/m/index.js]: It is better to use `p=ej` the way to use this command is the following:

`r2.cmd('p=ej', (x) => ...)`

the argument for `p=ej` specifies in how many blocks you want to split the entire area analyzed. to specify the length just use the second parameter and run this command in a different offset. So the correct command will be:

`p=ej 20 $s @ $M`

note that '20' is the number of rows you will get in the resulting JSON.

Comment (radare) [inline www/m/index.js]: use single quotes `'` instead of `"`

File: www/m/index.js [hunk 3]
```diff
@@ -1091,6 +1090,34 @@ function updateInfo() {
...
+		var lines = d.split(/\n/g);
+		var body = "";
+		var values = new Array();
+
+		for (var i in lines) {
```
Comment (radare) [inline www/m/index.js]: using cmdj you'll get the object so parsing will be not necessary

**Comment by radare** [comment]
ping

**Comment by iriahi** [comment]
Here is the updated version.

File: www/m/index.html (hunk 1 above)
Comment (iriahi) [inline www/m/index.html]: Done :)
But this template is not used by the entropy chart anymore. The entropy chart is inline svg.

File: www/m/index.js (hunk 2 above)
Comment (iriahi) [inline www/m/index.js]: I'm now using p=ej with 50 blocks (the chart width is 500px so that's why).
I've also changed double quotes to single ones.

**Comment by iriahi** [comment]
It is now passing build.

**Pull request merged**